pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -2242,12 +2252,11 @@ func buildListener(opts buildListenerOpts) *xdsapi.Listener {
 	return listener
 }
 
-// appendListenerFallthroughRoute adds a filter that will match all traffic and direct to the
-// PassthroughCluster. This should be appended as the final filter or it will mask the others.
-// This allows external https traffic, even when port the port (usually 443) is in use by another service.
-func appendListenerFallthroughRoute(l *xdsapi.Listener, opts *buildListenerOpts,
-	node *model.Proxy, currentListenerEntry *outboundListenerEntry) {
-	wildcardMatch := &listener.FilterChainMatch{}
+// Create pass through filters for the listener assuming all the other filter chains are ready.
+// The match member of pass through filter chain depends on the existing non-passthrough filter chain.
+// TODO: Calculate the filter chain match to replace the wildcard and replace appendListenerFallthroughRoute.
+func (configgen *ConfigGeneratorImpl) appendListenerFallthroughRouteForCompleteListener(l *xdsapi.Listener, node *model.Proxy, push *model.PushContext) {
```

Comment:
may lack some context, why it's called route? you mean the route of filter chain? if so, i feel this can cause some confusion since route has overloaded meaning.

Reply:
Good question. I don't know. The name is borrowed from deleted function `appendListenerFallthroughRoute`

I am pretty sure it's not the `Route` in RDS